run_linux.go

```diff
@@ -2850,6 +2854,18 @@ func (b *Builder) cleanupRunMounts(context *imagetypes.SystemContext, mountpoint
 			prevErr = err
 		}
 	}
+	// unlock if any locked files from this RUN statement
+	for _, path := range artifacts.LockedTargets {
+		if _, err := os.Stat(path); err == nil {
+			lockfile, err := lockfile.GetLockfile(path)
+			if err != nil {
+				return err
```

Comment:
Should this be returned or logged as prevErr and continue unlocking other lockedtargets?